src/ex4/server/services/utils.js

```diff
@@ -0,0 +1,35 @@
+import { promises as fs } from "fs";
+import { dirname, join } from "path";
+import { fileURLToPath } from "url";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const tasksFile = join(__dirname, "..", "data", "tasks.json");
+
+async function readTasksFile() {
+  try {
+    const data = await fs.readFile(tasksFile);
+    return JSON.parse(data.toString());
+  } catch (error) {
+    console.error(`Got an error trying to read the file: ${error.message}`);
```

Comment:
better throw and error and reflect it the user that something has failed from being done, maybe alert them